### PR TITLE
Using multiple where conditions in a list in single where method 

### PR DESCRIPTION
**Pull request by mathankumart**
Using multiple where conditions in a list in single where method as eloquent usage (Issue #103 )

### CODE REVIEW

File: orator/query/builder.py [hunk 1]
```diff
@@ -353,6 +353,15 @@ def where(self, column, operator=Null(), value=None, boolean='and'):
         if isinstance(column, QueryBuilder):
             return self.where_nested(column, boolean)
 
+        if isinstance(column, list):
+            nested = self.new_query()
+            for condition in column:
+                if isinstance(condition, list) and len(condition) == 3:
+                    nested.where(condition[0], condition[1], condition[2])
+                else:
+                    raise ArgumentError("Invalid inputs in the list")
```
Comment (sdispater) [inline orator/query/builder.py]: I think the error message here should more explicit.

Something like "Invalid conditions in where() clause.".

File: tests/query/test_query_builder.py [hunk 2]
```diff
@@ -683,6 +683,15 @@ def test_where_shortcut(self):
...
 
+    def test_multiple_wheres_in_list(self):
+        builder = self.get_builder()
+        builder.select('*').from_('users').where([['name', '=', 'bar'], ['age', '=', 25]])
+        self.assertEqual(
+            'SELECT * FROM "users" WHERE ("name" = ? AND "age" = ?)',
+            builder.to_sql()
+        )
+        self.assertEqual(['bar', 25], builder.get_bindings())
+
```
Comment (sdispater) [inline tests/query/test_query_builder.py]: We should also test the case where conditions are invalid, to see if the exception is actually raised.

Comment (mathankumart) [inline tests/query/test_query_builder.py]: Added the test cases for exceptions :+1:

**Comment by sdispater** [comment]
Thanks!

**Pull request merged**